var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -368,6 +368,21 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
 
     # ###################### Dependencies ##########################
 
+    # External Kokkos
+    depends_on("kokkos@4.1.00", when="@14.4.0: +kokkos")
+    depends_on("kokkos +wrapper", when="trilinos@14.4.0: +kokkos +wrapper")
+    depends_on("kokkos ~wrapper", when="trilinos@14.4.0: +kokkos ~wrapper")
```

Comment:
I was not aware that Trilinos could even build with Kokkos without using nvcc_wrapper as the underlying C++ compiler.  Is that true?  How long has that been the case?

Reply:
I've never done it, but my understanding is if you have a cuda compatible clang compiler you do not need nvcc_wrapper.